components/ILIAS/COPage/PC/Login/class.ilPCLoginPageElement.php

```diff
@@ -26,7 +26,6 @@ class ilPCLoginPageElement extends ilPageContent
 {
     private static array $types = array(
         'login-form' => 'login_form',
-        'cas-login-form' => 'cas_login_form',
```

Comment:
Hi @alex40724 , we had to change the array values here to remove CAS competely from our code base. If you have any objections, please leave a comment.